app/page.tsx

```diff
@@ -74,10 +76,15 @@ const StyledGreen = styled.div`
   color: #468259;
 `;
 
-const StyledSnowManAndTree = styled(Image)`
+const StyledSnowManTree = styled(Image)`
   position: absolute;
   width: 188px;
   height: 175px;
   bottom: 50px;
   right: 2px;
 `;
+
+const StyledImageWrapper = styled(motion.div)`
+  position: absolute;
```

Comment:
글고 여기 스타일 쓰인 부분도 m으로 import해서 사용해야될 것 같아용 ~~

```suggestion
const StyledImageWrapper = styled(m.div)`
  position: absolute;
```

Reply:
넵 수정했습니다!